addons/website_slides/static/src/scss/website_slides.scss

```diff
@@ -421,7 +421,7 @@ $line-height-truncate: 1.25em;
 
         .o_wslides_lesson_nav {
             .navbar {
-                @if ( not has-enough-contrast($body-bg, $dark) ) {
+                @if ( lightness($body-bg) < 50% ) {
```

Comment:
In the fix commit you mention, they are setting the colors based on the result of this lightness alone, but here we are still using the other theme colors, which makes me suspect that there could be other theme configuration that would still not work after this...

Or am I wrong?

Reply:
I have tested with a few color combination and it seems like the navbar is always visible
What color combination would you be thinking about ?

Reply:
Shouldn't we just use the same logic as is done for the original navbar styling? i.e. something like `color-contrast($body-bg) != $color-contrast-dark` (and maybe check in 18/master if it changed to match it again)

It seems to work ok and in all likelihood it should work just as well the website navbar does at least as far as ensuring contrast goes.

It's a bit odd to have to "override" this manually anyway imo but I'm not skilled enough in sass (or css) to suggest a good fix